lib/LocalVerifiableCredentialStore.js

```diff
@@ -34,6 +34,7 @@ export class LocalVerifiableCredentialStore {
           'meta.profileId',
           'meta.displayable',
           'meta.parentId',
+          'content.type'
```

Comment:
I have all the same concerns here about indexing again. I see that the find function allows for `parentId` to be omitted and `type` to be present, for example. That would not result in the index being used properly.

What are the queries that are meant to be supported here? Which fields will be present? The indexes must match those fields. If we need another index for the queries we need to support we should add it.